image-loader/conf/logger.xml

```diff
@@ -23,6 +23,9 @@
 
     <logger name="play" level="INFO" />
     <logger name="application" level="DEBUG" />
+    <logger name="com.amazonaws">
+        <level value="ERROR" />
+    </logger>
```

Comment:
Not sure. Have switched the root level to debug in DEV and the logs are not any louder. Will change.